Server/src/bgp/linkstate/MPLinkStateAttr.cpp

```diff
@@ -145,20 +166,7 @@ namespace bgp_msg {
                              peer_addr.c_str(), len);
                 }
 
-                std::string flags = "";
-
-                if (*data & NODE_FLAG_MASK_OVERLOAD)
-                    flags += "O";
-                if (*data & NODE_FLAG_MASK_ATTACH)
-                    flags += "T";
-                if (*data & NODE_FLAG_MASK_EXTERNAL)
-                    flags += "E";
-                if (*data & NODE_FLAG_MASK_ABR)
-                    flags += "B";
-                if (*data & NODE_FLAG_MASK_ROUTER)
-                    flags += "R";
-                if (*data & NODE_FLAG_MASK_V6)
-                    flags += "V";
+                std::string flags = this->parse_flags_to_string(*data, std::array<std::string, 8>{"O", "T", "E", "B", "R", "V"});
```

Comment:
Is it ok to use smth like:
```
        static const std::string NODE_FLAG_MASK_OVERLOAD    = "O";
        static const std::string NODE_FLAG_MASK_ATTACH      = "T";
        static const std::string NODE_FLAG_MASK_EXTERNAL    = "E";
        static const std::string NODE_FLAG_MASK_ABR         = "B";
        static const std::string NODE_FLAG_MASK_ROUTER      = "R";
        static const std::string NODE_FLAG_MASK_V6          = "V";
```

in header file? I want to use String because flags with more then one char exist.

So my call will look like:
```
std::string flags = this->parse_flags_to_string(*data, std::array<std::string, 8>{
NODE_FLAG_MASK_OVERLOAD, 
NODE_FLAG_MASK_ATTACH, 
NODE_FLAG_MASK_EXTERNAL, 
NODE_FLAG_MASK_ABR, 
NODE_FLAG_MASK_ROUTER, 
NODE_FLAG_MASK_V6
});
```
But it's not clear for me why we need it. There are 4 places where I use parse_flags_to_string and each time flags are different. So each time it's needed to create such a group of strings. And this strings used only there and and nowhere else. Don't you think it's redundantly? I wanted to treat flags just like symbol sequence without any logical meaning inside collector.